rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -519,6 +477,10 @@ private int doEnd(boolean cancel) {
             status = toLRAStatus(res);
         }
 
+        // it would be less invasive to allow recoovery to run post LRA actions
```

Comment:
Perhaps the TCK could have an RFE raised?

Reply:
This PR runs agains MP-LRA RC1 but I notice that the latest snapshot does run a recovery pass now. Therefore I have reworded the comment.

Reply:
Thanks @mmusgrov I note the previous comment indicated invasiveness of calling this here but timeliness now? TBH I don't quite understand why they are considered "Post" actions as it seems to be the phase that actually completes the LRA?

Reply:
Invasive in the sense that it bulks up the number of changes in the PR. I changed it to timeliness since it is a better functional argument for it.

Reply:
The are called "Post LRA" actions since they are performed after the LRA has reached a terminal state.